js/components/report/iframe-answer.js

```diff
@@ -112,8 +113,9 @@ export default class IframeAnswer extends PureComponent {
     return (
       <div className={`iframe-answer ${responsive ? "responsive" : ""}`} data-cy="iframe-answer">
         <div className={`iframe-answer-header ${responsive ? "responsive" : ""}`}>
-          { answerText && <div>{ answerText }</div> }
-          { !alwaysOpen && this.renderLink() }
+          { answerText && <div>{ renderHTML(answerText) }</div> }
+          {/* This assumes only scaffolded questions and fill in the blank questions have answerTexts */}
+          { (!answerText && (!alwaysOpen && this.renderLink())) }
```

Comment:
Does this change the behavior on the previous uses of this component?  Before we had:
`{ !alwaysOpen && this.renderLink() }`
Now we have:
`{ (!answerText && (!alwaysOpen && this.renderLink())) }`

Reply:
Ah, I think I get it now.  Doesn't seem to affect existing answers.  You might want to refactor this to be a little clearer with:
```
          { answerText ?
            <div>{ renderHTML(answerText) }</div>
            : !alwaysOpen && this.renderLink() /* This assumes only scaffolded questions and fill in the blank questions have answerTexts */
          }
```

Reply:
That's what I did at first but then realized that the mock data that was originally there had and answerText for the interactives (cypress tests started failing for the old reports), so I put it back in and made the logic to whether or not display the link separate.

Reply:
Huh, I would think it would do the same thing.

Reply:
We have `answerText` and `!answerText` - that should work in a ternary.

Reply:
Unless it's a subtlety about JS negation that I'm missing.

Reply:
If it truly makes the tests fail to add my code, then I would leave it.  I would at least double check it.